produce_snp_matrix.py

```diff
@@ -0,0 +1,44 @@
+from glob import glob
+import subprocess
+import argparse
+import os
+
+def snps(input_path, output_path):
+    # glob filenames
+    megafasta = ""
+    paths = glob(input_path + "*_consensus.fas")
+    for path in paths:
+        file = open(path)
```

Comment:
use this instead: 
`with open(path) as file:
fasta = file.read()
` 
it basically automatically closes the file after it's been read

Reply:
Yeah also, not that important, but i think make sure you close the file afterwards